pylinalg/matrix.py

```diff
@@ -129,8 +127,9 @@ def mat_from_euler(angles, /, *, order="xyz", out=None, dtype=None):
     angles : ndarray, [3]
         The euler angles.
     order : string, optional
-        The order in which the rotations should be applied. Default
-        is "xyz".
+        The rotation order as a string. Can include 'X', 'Y', 'Z' for intrinsic
+        rotation (uppercase) or 'x', 'y', 'z' for extrinsic rotation (lowercase).
+        Default is "xyz".
```

Comment:
Out of curiosity, can a user also pass (or think he can pass) a mix of lowercase and uppercase?

Reply:
Yeah they can but it will not give well defined results. I could validate inputs better.